examples/s3-update-example-cli/package.json

```diff
@@ -75,10 +75,9 @@
   "private": true,
   "repository": "jdxcode/s3-update-example-cli",
   "scripts": {
-    "postpack": "rm -f oclif.manifest.json",
-    "posttest": "tsc -p test --noEmit && tslint -p test -t stylish",
```

Comment:
`posttest` got dropped

Reply:
It's still there. I think it was below this line somewhere